src/content/docs/pt-br/guides/cms/wordpress.mdx

```diff
@@ -0,0 +1,193 @@
+- package.json
+</FileTree>
+
+A chamada para a API retorna u objeto que inclui as propriedades:
+
+- `title.rendered` - Contém o HTML do título do post.
+- `content.rendered` - Contém o HTML do conteúdo do post.
+- `slug` - Contém o slug do post. (Isso fornece o link para as páginas de dinossauro individuais geradas dinamicamente.)
+
+```astro title="/src/pages/index.astro"
+---
+import Layout from "../layouts/Layout.astro";
+
+let res = await fetch("https://norian.studio/wp-json/wp/v2/dinos");
+let posts = await res.json();
+---
+<Layout title="Dinos!">
+  <section>
+    <h1>Lista de Dinossauros</h1>
+    {
+      posts.map((post) => (
+        <article>
+          <h2>
+            <a href={`/dinos/${post.slug}/`} set:html={post.title.rendered} />
+          </h2>
+          <Fragment set:html={post.content.rendered} />
+        </article>
+      ))
+    }
+  </section>
+</Layout>
+```
+
+### Usando a API WordPress para gerar páginas
+
+A página `src/pages/dinos/[slug].astro` [gera uma página dinamicamente](/pt-br/guides/routing/#rotas-dinâmicas) para cada dinossauro.
+
+```astro title="/src/pages/dinos/[slug].astro"
+---
+import Layout from '../../layouts/Layout.astro';
+
+const { slug } = Astro.params;
+
+let res = await fetch(`https://norian.studio/wp-json/wp/v2/dinos?slug=${slug}`);
+let [post] = await res.json();
+
+// getStaticPaths() é exigido para sites Astro estáticos.
+// Se usar SSR, você não precisará dessa função.
+export async function getStaticPaths() {
+  let data = await fetch("https://norian.studio/wp-json/wp/v2/dinos");
+  let posts = await data.json();
+
+  return posts.map((post) => ({
+    params: { slug: post.slug },
+    props: { post: post },
+  }));
+}
+---
+<Layout title={post.title.rendered}>
+  <article>
+    <h1 set:html={post.title.rendered} />
+    <Fragment set:html={post.content.rendered} />
+  </article>
+</Layout>
+```
+
+### Retornando recursos embutidos
+
+O parâmetro `_embed` instrui o servidor a retornar recursos (embutidos) relacionados.
+```astro title="src/pages/dinos/[slug].astro" /&_embed/
+---
+const { slug } = Astro.params;
+
+let res = await fetch(`https://norian.studio/wp-json/wp/v2/dinos?slug=${slug}&_embed`);
+let [post] = await res.json();
+---
+```
+
+A propriedade `_embedded['wp:featuredmedia']['0'].media_details.sizes.medium.source_url` é retornada, e pode ser usada para mostrar a imagem destacada em cada página de dinossauro. (Substitua `medium` com o tamanho de imagem que deseja.)
+
+```astro title="/src/pages/dinos/[slug].astro" {3}
+<Layout title={post.title.rendered}>
+  <article>
+    <img src={post._embedded['wp:featuredmedia']['0'].media_details.sizes.medium.source_url} />
+    <h1 set:html={post.title.rendered} />
+    <Fragment set:html={post.content.rendered} />
+  </article>
+</Layout>
+```
+
+### Publicando seu site
+
+Para lançar seu site visite o nosso [guia de lançamento](/pt-br/guides/deploy/) e siga as instruções para o seu fornecedor de hospedagem preferido.
```

Comment:
I would say "lançamento" doesn't work well in the context of "deploy" - all other pages in Portuguese have maintained the original term, and I would vouch for doing the same here.
```suggestion
Para lançar seu site visite o nosso [guia de deploy](/pt-br/guides/deploy/) e siga as instruções para o seu fornecedor de hospedagem preferido.
```